Bio/SeqUtils/__init__.py

```diff
@@ -409,7 +410,7 @@ def six_frame_translations(seq, genetic_code=1):
     M  A  I  V  M  G  R  *
     auggccauuguaaugggccgcuga   54 %
     uaccgguaacauuacccggcgacu
-    A  M  T  I  P  R  Q 
+    A  M  T  I  P  R  Q
```

Comment:
You (or more likely your editor) has accidentally removed a trailing space here, which broke the doctest.

Reply:
Thanks for catching that!